app/src/androidTest/java/org/simple/clinic/TestData.kt

```diff
@@ -602,7 +602,7 @@ class TestData @Inject constructor(
       uuid: UUID = UUID.randomUUID(),
```

Comment:
`repository` is getting truncated in this commit message. Let's remove `from repository` entirely? It's already obvious that all data come from repositories.